tests/libres_tests/res/enkf/test_res_config.py

```diff
@@ -349,6 +349,7 @@ def test_res_config_dict_constructor(setup_case):
     # add a missing entries to config file
     with open("user_config.ert", "a+") as ert_file:
         ert_file.write("JOB_SCRIPT ../../../script.sh\n")
+        ert_file.write("NUM_CPU 0\n")
```

Comment:
This entry is optional. Why do we need to add it?

Reply:
because in the config dict, it's set to zero, and the config file path sets that value to 1 if it's not given. so it's to make the test pass.

Reply:
Don't we want the config_dict path to also set it to 1 then?